app/lib/notifications.server.ts

```diff
@@ -4,27 +4,28 @@ import logger from './logger.server';
 
 import type { SendMailOptions } from 'nodemailer';
 
-const { NOTIFICATIONS_EMAIL_USER, NODE_ENV, MAILHOG_SMTP_PORT } = process.env;
-const { NODEMAILER_CLIENT_ID, NODEMAILER_ACCESS_TOKEN } = secrets ?? {};
+const { NOTIFICATIONS_EMAIL_USER, NODE_ENV, MAILHOG_SMTP_PORT, NOTIFICATIONS_SMTP_PORT } =
+  process.env;
+const { NOTIFICATIONS_USERNAME, NOTIFICATIONS_PASSWORD } = secrets ?? {};
 
 const initializeTransport = () => {
   if (!NOTIFICATIONS_EMAIL_USER) {
     throw new Error('Missing Nodemailer user. Skipping nodemailer configuration');
   }
   if (NODE_ENV === 'production') {
-    if (!NODEMAILER_CLIENT_ID || !NODEMAILER_ACCESS_TOKEN) {
+    if (!NOTIFICATIONS_USERNAME || !NOTIFICATIONS_PASSWORD) {
       throw new Error(
         'Missing Nodemailer environment variables or docker secrets. Skipping nodemailer configuration'
```

Comment:
I would improve this error message: tell them which variables are actually missing, so it's easier to debug startup crashes.